Make sure we keep all the prototypes the same or we hit a build
failure like this:
converter/ppm/ppmtompeg/headers/jpeg.h:47:
error: previous declaration of 'JMovie2JPEG' was here
nojpeg.c:89: error: conflicting types for 'JMovie2JPEG'

http://bugs.gentoo.org/111895

--- converter/ppm/ppmtompeg/headers/jpeg.h
+++ converter/ppm/ppmtompeg/headers/jpeg.h
@@ -44,6 +44,6 @@
 #include "ansi.h"
 
 
-extern void     JMovie2JPEG _ANSI_ARGS_((char *infilename, char *obase,
-                                        int start, int end));
-extern void ReadJPEG _ANSI_ARGS_((MpegFrame * mf, FILE *fp));
+extern void     JMovie2JPEG _ANSI_ARGS_((const char * const infilename, const char * const obase,
+                                        int const start, int end));
+extern void ReadJPEG _ANSI_ARGS_((MpegFrame * const mf, FILE * const fp));
--- converter/ppm/ppmtompeg/nojpeg.c
+++ converter/ppm/ppmtompeg/nojpeg.c
@@ -86,7 +86,7 @@
 JMovie2JPEG(const char * const infilename,
             const char * const obase,
             int          const start,
-            int          const end) {
+            int          end) {
 
     pm_error("This program has not been built with the "
              "ability to handle JPEG input files");
--- converter/ppm/ppmtompeg/jpeg.c
+++ converter/ppm/ppmtompeg/jpeg.c
@@ -72,10 +72,10 @@
  ************************************************************************/
 void
   JMovie2JPEG(infilename,obase,start,end)
-char *infilename;       /* input filename string */
-char *obase;            /* output filename base string=>obase##.jpg */ 
-int start;              /* first frame to be extracted */
-int end;                /* last frame to be extracted */
+const char * const infilename;  /* input filename string */
+const char * const obase;       /* output filename base string=>obase##.jpg */ 
+int const start;                /* first frame to be extracted */
+int end;                        /* last frame to be extracted */
 {
   FILE *inFile;			/* Jmovie file pointer */
   FILE *outFile;		/* JPEG file pointer for output file */
